### DISCUSSION TITLE
Linuxbrew has gcc@10.2 as alias to gcc@11

### DISCUSSION THREAD

**Discussion by astiob** [Homebrew/discussions#2282] [Linux] [answered]
I don’t have permission to open a pull request nor an issue on `linuxbrew-core`, but this is a Linuxbrew-only issue: when GCC 10.2 was merged from `homebrew-core`, a `gcc@10.2` alias was created; but when GCC was upgraded to version 11, the alias wasn’t removed. It doesn’t exist in `homebrew-core`.

**Comment by carlocab** [comment]
@Homebrew/linux

**Comment by danielnachun** [comment, marked as answer]
Thanks for reporting this.  Fixed by https://github.com/Homebrew/linuxbrew-core/pull/24547.
